### ISSUE TITLE
If I want to run for input size 128x128

### ISSUE THREAD

**Issue by coderTazz** [Jongchan/tensorflow-vdsr#13]
If I wanted to run VDSR (with adam.. without gradient clipping and all) for input size of 128x128 to get a super resolved output with scale 2 (256x256).
How should I go about it?
I mean, can someone give me an idea about the number of epochs, weightDecay, momentum values and learning rates required?

**Comment by Jongchan** [comment]
The super-resolution CNN are trained with small patches(from 30 * 30~48 * 48), but can be deployed on larger input images (because the network is fully-convolutional, can handle any input size).
So, the solution is to train your network using the provided code, and change the input size during deploy(test) phase according to your need.

ps. I think I might have to add some examples, where input is an image file.. for future usage..
